src/app/pages/authentication/login/login.component.css

```diff
@@ -0,0 +1,68 @@
+.login-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  background-image: url("../../../../assets/images/airline_image1.jpg");
```

Comment:
don't use ../ before assets. you can directly call assets/images/...png

Reply:
@aky-wtag vai resolved